code/game/gamemodes/clock_cult/clock_effects/reebe_rift.dm

```diff
@@ -0,0 +1,79 @@
+//A two-way portal that leads to and from the City of Cogs. These spawn across the station and lead to one location.
+//This file also contains the City of Cogs beckoner.
+/obj/effect/clockwork/reebe_rift
+	name = "spatial rift"
+	desc = "A shimmering portal to another area. You can see shimmers of brassy colors on the other side."
+	clockwork_desc = "A two-way rift that leads between the station and the Celestial Derelict."
+	icon_state = "spatial_gateway"
+	density = 1
+	light_range = 2
+	light_power = 3
+	light_color = "#6A4D2F"
+
+/obj/effect/clockwork/reebe_rift/Initialize()
+	if(!GLOB.city_of_cogs_beckoner)
+		return INITIALIZE_HINT_QDEL
```

Comment:
Actually, while you're around, when should I use . = ..() instead of ..()? I've asked before but the only answers I've gotten are "you should know" and unhelpful ones.